src/models/course/index.ts

```diff
@@ -16,6 +16,8 @@ export interface Course {
   id: string
   tagline?: string
   description?: string
+  logo?: string
+  stickerPrice?: number
```

Comment:
should be markedPrice here not stickerPrice 

rest looks good

Reply:
Done !